src/taskList.dart

```diff
@@ -0,0 +1,45 @@
+import 'task.dart';
+import 'dart:io';
+
+class TaskList {
+  List<Task> _tasks = [];
+  TaskList(this._tasks);
+  Task operator [](int taskID) {
+    return _tasks[taskID];
+  }
```

Comment:
smart way to add an operator to the taskList. Didnt even know that was possible 😂